CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.5.1 (Next)
 
+* [#33](https://github.com/dblock/slack-ruby-bot/pull/33): `SlackRubyBot::App.instance` now creates an instance of the class on which it is called.
```

Comment:
Make this look like all the other changelog lines with a link to your Github profile.